+bids/layout.m

```diff
@@ -286,6 +286,7 @@ function tolerant_message(tolerant, msg)
             '(?<ses>_ses-[a-zA-Z0-9]+)?' ...     % ses-<label>
             '(?<acq>_acq-[a-zA-Z0-9]+)?' ...     % acq-<label>
             '(?<rec>_rec-[a-zA-Z0-9]+)?' ...     % rec-<label>
+            '(?<dir>_dir-[a-zA-Z0-9]+)?' ...     % dir-<label>
```

Comment:
Given that the parsing is based on the bids schema that mentions the dir entity, this should be fine.

Reply:
Ok